app/forms/webauthn_setup_form.rb

```diff
@@ -52,6 +54,7 @@ def consume_parameters(params)
     @client_data_json = params[:client_data_json]
     @name = params[:name]
     @platform_authenticator = (params[:platform_authenticator].to_s == 'true')
+    @transports = params[:transports]&.split(',')
```

Comment:
should we add a default value so it's always an array?
```suggestion
    @transports = params[:transports]&.split(',') || []
```

Reply:
Yeah, I'm flip-flopping a little bit whether to allow an empty array, vs. considering that invalid, vs. explicitly setting to `nil`. It'll become more relevant for the authentication bits, since we need to support existing configurations, where `transports` acts as a filter of sorts, and the difference between `nil` and `[]` is especially meaningful.